core/crypto/blake2/blake2b160.cpp

```diff
@@ -147,14 +147,14 @@ namespace fc::crypto::blake2b {
 
     Ctx ctx{BLAKE2B512_HASH_LENGTH};
 
-    const int one_kb = 1024;
-    std::string bytes(one_kb, ' ');
-    file_stream.read(bytes.data(), one_kb);
+    const int buffer_size = 32 * 1024;
```

Comment:
Why do you need to increase file read buffer?

Reply:
the maximum file size is 32GB now